### ISSUE TITLE
Issues after refactoring 

### ISSUE THREAD

**Issue by lego963** [opentelekomcloud/terraform-provider-opentelekomcloud#1288]
We have to re-run all acceptance test.

# Terraform Version
Provider `v1.25.0`

# Affected Resource(s)
All `resources` and `data_sources`

# References
#1287
#1283
#1303

**Comment by Unichron** [comment]
It's a bit worrying that this level of breakage was not caught automatically.

**Comment by outcatcher** [comment]
@Unichron Sorry for the inconvenience, we're trying to make the provider more stable and tested but have taken a bit of a harsh turn here. The change behind those problems is actually one step forward to making the provider more tested ― what is crashing now was just silently not working before.
We are yet to automate all acceptance tests running, but that was still our mistake not testing enough such a big change. I hope we will avoid such problems in the future.

**Comment by DanielHabenicht** [comment]
Might this also be part of that bug? If so I can also create an issue or should this also be fixed with 1.25.1?
```
╷
│ Error: 4 errors occurred:
│ 	* Invalid address to set: []string{"address"}
│ 	* Invalid address to set: []string{"fixed_ip"}
│ 	* Invalid address to set: []string{"tenant_id"}
│ 	* Invalid address to set: []string{"pool"}
│ 
│ 
│ 
│   with module.cce.opentelekomcloud_networking_floatingip_associate_v2.associate,
│   on modules/cce/main.tf line 108, in resource "opentelekomcloud_networking_floatingip_associate_v2" "associate":
│  108: resource "opentelekomcloud_networking_floatingip_associate_v2" "associate" {
│ 
╵
```

**Comment by outcatcher** [comment]
@DanielHabenicht this really seems like that issue, will check it.

**Comment by lego963** [comment]
All issues have been fixed in `v1.25.4`
